### ISSUE TITLE
input_number slider falsely introduces many decimals

### ISSUE THREAD

**Issue by krissen** [home-assistant/home-assistant-polymer#1687]
**Home Assistant release with the issue:** 0.78.0, but present since at least 0.76.0

**Browser and Operating System:** Chrome @ Linux, iOS, and macOS. Homeassistant app @ iOS and macOS.

**Description of problem:**
I have five input_number slides with largely identical configuration; the following are no. 2 and 3:
```
slider_ljusniva_2:
  name:"Ljusnivå 2 om under"
  min:-5
  max:5
  step:0.1
  mode:'slider'

slider_ljusniva_3:
  name:"Ljusnivå 3 om under"
  min:-5
  max:5
  step:0.1
  mode:'slider'
```

input_number sliders 3, 4 and 5 work as expected, but sliders 1 and 2 are shown with too many decimals, as seen below:

![96ad5442-5237-4829-82bc-b12e0c888a27](https://user-images.githubusercontent.com/2943684/45804732-30e5af80-bcbd-11e8-8ba4-ace3872937e1.jpeg)

I have tried setting the variable for 1 and 2 to something with only one decimal in dev-states, but when frontview loaded, additional decimals had again been added.

**Comment by balloob** [comment]
What is the state in the state machine?

**Comment by krissen** [comment]
> What is the state in the state machine?

In short: correct. One decimal.

![skarmklipp 2018-09-21 13 16 47](https://user-images.githubusercontent.com/2943684/45878266-b8581f00-bda0-11e8-9cf1-ac83101af0b3.png)

![skarmklipp 2018-09-21 13 17 04](https://user-images.githubusercontent.com/2943684/45878274-bc843c80-bda0-11e8-9759-383c7ac308de.png)

**Comment by krissen** [comment]
Ah; now I see why it was only once decimal in dev-states; I had manually set it as such, there. 

_The variable in dev-states is what the slider sets it to._ I just set the slider as shown in the first image below. Having done that, it looks as follows in dev-states. Do note that the second one is without decimals. In other words; the variable is set to what the slider shows. (Why it was different in the first screenshots I uploaded had to do with me manually setting it, I'm sure. Sorry for the confusion.)

![skarmklipp 2018-09-21 13 29 03](https://user-images.githubusercontent.com/2943684/45878767-92337e80-bda2-11e8-95b0-6cc6ae13e76b.png)

![skarmklipp 2018-09-21 13 29 17](https://user-images.githubusercontent.com/2943684/45878775-9790c900-bda2-11e8-8504-cc14bfc3c590.png)
